monitor/filter_exception_decoder.py

```
@@ -16,30 +16,81 @@
+
+    # Chip name mapping for ROM ELF files
+    CHIP_NAME_MAP = {
+        "esp32": "esp32",
+        "esp32s2": "esp32s2",
+        "esp32s3": "esp32s3",
+        "esp32c2": "esp32c2",
+        "esp32c3": "esp32c3",
+        "esp32c6": "esp32c6",
+        "esp32h2": "esp32h2",
+        "esp32p4": "esp32p4",
+    }
```

Ruff (0.14.0)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)